src/components/index.js

```diff
@@ -90,3 +90,5 @@ export {
   useGridKeyboardNavigationContext,
   GridKeyboardNavigationContext
 } from "./GridKeyboardNavigationContext/GridKeyboardNavigationContext";
+export { default as Badge } from "./Badge/Badge";
+export { default as Indicator } from "./Indicator/Indicator";
```

Comment:
why do we actually extract this into two different components?

Reply:
+1